File: Math/Easy/509/509.c
int fib(int n){
    if(n < 2){
        return n;
    }
    int i = 0, j = 1;
    int k = j;
    for(int idx = 2; idx < n; idx++){
        k = i + j;
        i = j;
        j = k;
    }
    return j;
}

File: Math/Easy/509/509.js
/**
 * @param {number} n
 * @return {number}
 */
var fib = function(n) {
    if(n < 2){
        return n
    }
    let i = 0
    let j = 1
    let k = 0
    for(let idx = 0; idx < n - 1; idx++){
        k = i + j
        i = j
        j = k
    }
    return j
};